### DISCUSSION TITLE
How do I return Cache-Control headers for specific server/api

### DISCUSSION THREAD

**Discussion by msonowal** [nuxt/nuxt#19402] [Questions (Nuxt 2)] [answered]
I have defined the below in `/server/api/services/test` so I want to levarage browser cache by sending cache header on response where value is based on some condition say sometimes I want to set for 1 hour sometime 1 day etc

As by looking in the docs I am unable to find it.

export default defineEventHandler(async (event: H3Event) => {
    const { apiBase } = useRuntimeConfig()
    let url = `${apiBase}/test/services/${event.context.params.speciality}/test`

    const {data: service}: ApiResponse = await (await fetch(url)).json()
    return service
})

**Comment by Baroshem** [comment, marked as answer]
Hey,

What sbout using setResponseHeader from H3? https://www.jsdocs.io/package/h3#setResponseHeader

**Reply by Baroshem** [reply]
You should be able to set a header like `Cache-Control: max-age=604800` where Cache control is the name of the header and max-age is the value

**Reply by msonowal** [reply]
thanks man!
